draft-packages/tabs/KaizenDraft/Tabs/styles.scss

```diff
@@ -69,18 +60,34 @@
       }
     }
   }
-
-  &:hover {
-    @extend %tabHover;
-  }
 }
 
-.tab + .tab {
-  @include ca-margin($start: $ca-grid * 0.5);
+.row {
```

Comment:
the naming threw me off a bit. I thought it was the entire row for row-style-tabs, but it's actually a single tab. Suggest `rowTab` and `columnTab`?